### ISSUE TITLE
CJK contents does not display correctly in termui

### ISSUE THREAD

**Issue by ProgramFan** [MichaelMure/git-bug#85]
When I add issues in Chinese, the characters are not displayed correctly in termui. For example, I get
<img width="939" alt="default" src="https://user-images.githubusercontent.com/32396/50376124-d05f6900-0643-11e9-937c-a270a4cfa12f.png">
for
<img width="939" alt="default" src="https://user-images.githubusercontent.com/32396/50376132-eb31dd80-0643-11e9-8595-f557ee346242.png">

**Comment by ProgramFan** [comment]
This issue shall be caused by the inability of gocui to handle unicode texts correctly, as suggested in https://github.com/jroimartin/gocui/issues/157. The nice thing is that somone provides a solution in https://github.com/jroimartin/gocui/issues/157. Would it possible to fix this in go-bug?

**Comment by MichaelMure** [comment]
I looked at this, and it's actually a problem in git-bug, not in gocui. Working on a fix.

**Comment by MichaelMure** [comment]
Could you try the latest revision from git to see if it's fixed ?

**Comment by ProgramFan** [comment]
Still not working, why?
<img width="939" alt="default" src="https://user-images.githubusercontent.com/32396/50400003-60c1b900-07be-11e9-96f3-750d51a2e9c0.png">

**Comment by ProgramFan** [comment]
I found that the Chinese characters are skipped interleaved. Like the above example:

改**进** LibFind 支**持** Targets，**可**更**方**便**地**支**持**二**维**三**维**混**用**、**编**译**参**数**依**赖

The ones in bold are skipped. The English chars are kept.

**Comment by MichaelMure** [comment]
Indeed, more fixes are needed. What happen is that the Chinese characters actually take **two** column in a terminal instead of always one as with the latin alphabet. It breaks an algorithm in git-bug, but it seems it also break gocui itself.

There is a PR with a fix in gocui (https://github.com/jroimartin/gocui/pull/173) but the project seems unmaintained for some month now.... That means I would need to fork gocui and maintain it ...

**Comment by ProgramFan** [comment]
After looking into the PR, I step into the fork from lazygit authors: https://github.com/jesseduffield/gocui, which seems to have fixed the problems (also 28+ commits newer than original gocui). This fork is used in lazygit, and it works:
<img width="939" alt="default" src="https://user-images.githubusercontent.com/32396/50407791-289b9400-0819-11e9-9ff3-f565c4326064.png">

**Comment by ProgramFan** [comment]
I tried to replace the gocui module in go-bug, but it seems lazygit changed the calling signature.
```
termui/label_select.go:119:22: not enough arguments in call to g.SetView
	have (string, int, int, int, int)
	want (string, int, int, int, int, byte)
```

**Comment by ProgramFan** [comment]
I managed to replace gocui with that from lazygit and CJK contents are displayed correctly in termui. I am still fixing the block wrap and will submit a PR after that.
<img width="939" alt="default" src="https://user-images.githubusercontent.com/32396/50446456-f6d91900-094f-11e9-93d3-d50342846afe.png">

**Comment by ProgramFan** [comment]
I just found your new commits handling these issues. Do you still want the PR?

**Comment by MichaelMure** [comment]
There is still `LeftPadMaxLine` to be fixed but it seems you have done it ? Could you do a PR for it ?

**Comment by MichaelMure** [comment]
Fixed with #86
